bindings/go/slangroom.go

```diff
@@ -15,7 +15,16 @@ type SlangResult struct {
 	Logs   string
 }
 
-func SlangroomExec(conf string, contract string, data string, keys string, extra string, context string) (SlangResult, error) {
+type SlangInput struct {
+	Conf     string
+	Contract string
+	Data     string
+	Keys     string
+	Extra    string
+	Context  string
+}
+
+func SlangroomExec(input SlangInput) (SlangResult, error) {
```

Comment:
Also prefer to refactor this to `Exec`... so we can call it as
`slangroom.Exec`